pkg/servicemesh/federation/status/manager_test.go

```diff
@@ -573,7 +573,7 @@ func TestStatusManager(t *testing.T) {
 			},
 			assertions: []func(t *testing.T, status *v1.ServiceMeshPeerStatus){
 				func(t *testing.T, status *v1.ServiceMeshPeerStatus) {
-					if status.DiscoveryStatus.Inactive[0].Watch.LastEvent.IsZero() {
+					if len(status.DiscoveryStatus.Inactive) > 0 && status.DiscoveryStatus.Inactive[0].Watch.LastEvent.IsZero() {
```

Comment:
I **think** this should be
```
if len(status.DiscoveryStatus.Inactive) == 0 || (len(status.DiscoveryStatus.Inactive) > 0 && status.DiscoveryStatus.Inactive[0].Watch.LastEvent.IsZero())
```

@rcernich what's your input here?

Reply:
Seems like `if len(status.DiscoveryStatus.Inactive) == 0 || status.DiscoveryStatus.Inactive[0].Watch.LastEvent.IsZero()` is enough.  I don't remember what that is doing, but I suspect if this panics, that's just a test failure, so all we're doing by cleaning things up is making sure the test doesn't crash?